### PR TITLE
Disabled 'stay alive' objective, reduced probabilities of steal and die

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Discussed the reasoning a bit in Discord, but with the current way rounds end these are kind of bad and really mess up the difficulty balance. Most objectives imply 'stay alive' and 'die' is easily cheesed by /suiciding 3 seconds before the round ends. These can be revisited after there's an actual evac shuttle and round end I think.

**Changelog**

:cl: Rane
- remove: Removed the objective to stay alive.
- tweak: Lowered max objective difficulty.
- tweak: Steal objective probability reduced by 1/3, and die objective probability reduced by 9/10.

### CODE REVIEW

**Review by moonheart08 — CHANGES_REQUESTED** [review]
Yea no, I want a p good argument for why these objectives should go. This needs discussed before merging.

**Comment by Elijahrane** [comment]
To summarize the arguments in Discord so far:
1. Most objectives basically imply "stay alive", so it's basically free difficulty knocked off if you get it because there's a 95% chance you redtext anyway if you die.
2. Die is very cheesable but it's a classic and usually gets more leeway on murderboning once servers start approaching MRP, so it should probably remain but be way rarer than it is now.

**Comment by metalgearsloth** [comment]
Don't see any arguments on github so gonna merge when CI passes.

File: Resources/Prototypes/Objectives/traitorObjectives..yml [hunk 1]
```diff
@@ -112,7 +120,8 @@
 - type: objective
   id: MagbootsStealObjective
   issuer: syndicate
-  difficultyOverride: 2.75
+  difficultyOverride: 1.75
```
Comment (metalgearsloth) [inline Resources/Prototypes/Objectives/traitorObjectives..yml:123]:
```suggestion
  difficultyOverride: 2.75
```

**Pull request merged**